components/kyverno/production/kflux-prd-rh03/kyverno-helm-values.yaml

```
@@ -0,0 +1,108 @@
+fullnameOverride: konflux-kyverno
+namespaceOverride: konflux-kyverno
+admissionController:
+  replicas: 3
+  initContainer:
+    securityContext:
+      allowPrivilegeEscalation: false
+      readOnlyRootFilesystem: true
+      runAsNonRoot: true
+      capabilities:
+        drop:
+        - "ALL"
+  container:
+    resources:
+      limits:
+        cpu: 500m
```

Maybe we can mirror what we already have on rh01/rh02?

```suggestion
      requests:
        cpu: 500m
        memory: 2Gi
      limits:
        cpu: 500m
        memory: 2Gi
```